marketplace/netlify-build/test/index.spec.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import NetlifyExtension from '../src/index.js';
+
+const pubsubMock = jest.fn();
+
+describe('NetlifyExtension', () => {
+  it('should render null if there are no users', () => {
```

Comment:
why's that?

Reply:
I removed this logic and also this test.